.github/workflows/builds.yml

```diff
@@ -6,9 +6,6 @@ on:
         - master
         - v1.*
   pull_request:
-    branches:
-        - master
-        - v1.*
```

Comment:
Does leaving this empty mean that it triggers on *all* PRs?

Reply:
Yeah, that is intended. Every PR deserves a build and automated validation. Are there reasons not to do so?

Reply:
I agree with you that we want a build on all PRs... my question is whether or not this was the correct GH action syntax to accomplish that

Reply:
Sorry, misunderstood. Yes, it triggers on all PRs whose activit type is one of opened, synchronize, or reopened.

> Note: More than one activity type triggers this event. For information about each activity type, see "[Webhook events and payloads](https://docs.github.com/en/github-ae@latest/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request)." By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. You can specify different activity types using the types keyword. For more information, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/github-ae@latest/articles/workflow-syntax-for-github-actions#onevent_nametypes)."